ci/ray_ci/tester.py

```diff
@@ -369,13 +369,16 @@ def _get_flaky_test_targets(
         # load flaky tests from yaml
         yaml_flaky_tests = set(yaml.safe_load(f)["flaky_tests"])
         # load flaky tests from DB
-        s3_flaky_tests = {
-            # remove "linux:" prefix for linux tests to be consistent with the
-            # interface supported in the yaml file
-            test.get_name().lstrip("linux:")
-            for test in Test.gen_from_s3(prefix=f"{operating_system}:")
-            if test.get_oncall() == team and test.get_state() == TestState.FLAKY
-        }
+        if os.environ.get("RAYCI_DISABLE_TEST_DB"):
```

Comment:
could you check the value to be `1` or `true`? otherwise, it will also be disabled when `RAYCI_DISABLE_TEST_DB=0`